lib/src/server.dart

```diff
@@ -52,6 +54,16 @@ class Server {
         const UserLogoutController(),
         middleware: [const AuthenticationMiddleware()],
       )
+      ..post(
+        'clock/in',
+        const ClockInController(),
+        middleware: [const ClockInMiddleware()],
+      )
+      ..post(
+        'clock/out',
```

Comment:
could we send the organizationId like this? https://github.com/rknell/alfred#routing--incoming-requests